test/MessageTraitTest.php

```diff
@@ -305,31 +305,6 @@ public function testNumericHeaderValues()
         ];
     }
 
-    /**
-     * @dataProvider testNumericHeaderValues
-     * @group 99
-     */
-    public function testFilterHeadersShouldAllowIntegersAndFloats($value)
```

Comment:
I'm concerned about removal of this test case, particularly as it was solving a reported issue. Can you reinstate it or rewrite it to test against the current changes, please?